packages/gestalt-design-tokens/config.json

```diff
@@ -50,6 +50,29 @@
           "options": {
             "fileHeader": "flowCustomHeader"
           }
+        },
+        {
+          "destination": "tokens_individual.js",
```

Comment:
this adds the ability for devs to import only tokens needed, which was a request in slack